reviewrot/__init__.py

```diff
@@ -123,28 +127,50 @@ def get_arguments(cli_arguments, config_arguments, config_mailer, choices):
         raise ValueError("Certificate file can't be used with insecure flag")
 
     format = parsed_arguments.get('format')
+    show_last_comment = parsed_arguments.get('show_last_comment')
     if (format in ['oneline', 'indented'] and
-            parsed_arguments.get('show_last_comment') is not None):
+            show_last_comment is not None):
         raise ValueError(
             '{} format doesn\'t support last comment functionality'.format(
                 format
             )
         )
 
+    irc = parsed_arguments.get('irc')
+
+    if irc and show_last_comment is not None:
+        raise ValueError(
+            'IRC output doesn\'t support last comment functionality'
```

Comment:
Why not?

Reply:
I am not sure I understand correctly, if you asking the reason why it is not supported it is because long output can be harder to read on irc